front/pages/api/login.ts

```diff
@@ -409,6 +409,12 @@ async function handler(
       const { error } = result;
 
       if (error instanceof AuthFlowError) {
+        logger.error(
```

Comment:
FYI `AuthFlowError` are Dust's specific errors (not related to Auth0):

https://github.com/dust-tt/dust/blob/c2a7dc0b03e82951c8a91fd7c3ac2dd1f91ad38f/front/lib/iam/errors.ts#L10-:15